DirectProgramming/C++SYCL_FPGA/include/README.md

```diff
@@ -25,6 +25,7 @@ This directory contains utility header libraries optimized for SYCL*-compliant F
 | `streaming_qri.hpp`                | QR-based inversion of matrices with pipe interfaces.       | `ReferenceDesigns/qri`
 | `streaming_cholesky.hpp`           | Cholesky decomposition of matrices with pipe interfaces.   | `ReferenceDesigns/cholesky`
 | `streaming_cholesky_inversion.hpp` | Cholesky-based inversion of matrices with pipe interfaces. | `ReferenceDesigns/cholesky_inversion`
+| `streaming_matmul.hpp` | Matrix multiplication with pipe interfaces. | `ReferenceDesigns/matmul`
```

Comment:
Please format the table and add a mention to the systolic array nature of the implementation